specs/grow-pg-spec.adoc

```diff
@@ -0,0 +1,99 @@
+= Grow PGs for a Ceph pool workflow by using tendrl-frontend UI
+
+The intent of grow PG workflow is to provide a mechanism to increase the PGs for one pool at a time in tendrl.
+
+* Assumption: 
+
+    - One or more Ceph cluster exists.
+
+    - One or more Ceph pools in the Ceph cluster exists.
+
+== Problem description
+
+This specification describes a mechanism to increase the PGs for one pool at a time in tendrl.
+
+== Use Cases
+
+WIP
+
+== Proposed change
+
+WIP
+
+=== Alternatives
+
+None
+
+=== Data model impact:
+
+=== Impacted Modules:
+
+==== Tendrl API impact:
+
+API to update pool -
+
+    * /api/1.0/8b0c637d-f5f7-40f6-acd9-50d5feb5f4b3/CephUpdatePool
+    * Request Body - 
+        '{"Pool.pool_id","Pool.poolname":"testPool","Pool.pg_num":200,"Pool.min_size":2}'
```

Comment:
Changing `poolname` and other parameters in one request is not allowed in backend. There is pre check and it would fail.
If you want to change `name`, pass that only with `pool_id` other parameters like `size`, `min_size` and `pg_num` can be changed separately.

Some sample curl requests are as below

```
curl -XPOST -d '{"Pool.pool_id":31, "Pool.poolname": "zest_pool"}' http://10.70.42.39:9292/1.0/d5f2afab-f188-45e8-9af6-8c587dca097f/CephUpdatePool
curl -XPOST -d '{"Pool.pool_id":31, "Pool.size": 2}' http://10.70.42.39:9292/1.0/d5f2afab-f188-45e8-9af6-8c587dca097f/CephUpdatePool
curl -XPOST -d '{"Pool.pool_id":31, "Pool.pg_num": 256}' http://10.70.42.39:9292/1.0/d5f2afab-f188-45e8-9af6-8c587dca097f/CephUpdatePool
curl -XPOST -d '{"Pool.pool_id": 31, "Pool.min_size": 2}' http://10.70.42.39:9292/1.0/d5f2afab-f188-45e8-9af6-8c587dca097f/CephUpdatePool
```